tools/shared.py

```diff
@@ -1561,7 +1560,8 @@ def nop(arg):
     env['RANLIB'] = quote(unsuffixed(EMRANLIB)) if not WINDOWS else 'python %s' % quote(EMRANLIB)
     env['EMMAKEN_COMPILER'] = quote(Building.COMPILER)
     env['EMSCRIPTEN_TOOLS'] = path_from_root('tools')
-    env['CFLAGS'] = env['EMMAKEN_CFLAGS'] = ' '.join(cflags)
+    if cflags:
+      env['CFLAGS'] = env['EMMAKEN_CFLAGS'] = ' '.join(cflags)
```

Comment:
Hmm. i wonder if we can remove EMMAKEN_CFLAGS completely?

Reply:
I think we use it in the test suite for some other things. But with some work we could get rid of it probably.